plugins/modules/jenkins_credential.py

```diff
@@ -0,0 +1,875 @@
+    inc_path:
+      - "include/path"
+      - "include/path2"
+    exc_path:
+      - "exclude/path"
+      - "exclude/path2"
+    inc_hostname:
+      - "included-hostname"
+      - "included-hostname2"
+    exc_hostname:
+      - "excluded-hostname"
+      - "excluded-hostname2"
+    schemes:
+      - "http"
+      - "https"
+    inc_hostname_port:
+      - "included-hostname:7000"
+      - "included-hostname2:7000"
+    exc_hostname_port:
+      - "excluded-hostname:7000"
+      - "excluded-hostname2:7000"
+
+- name: Add user_and_pass credential
+  community.general.jenkins_credential:
+    id: "userpass-id"
+    type: "user_and_pass"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    description: "User and password credential"
+    username: "user1"
+    password: "pass1"
+
+- name: Add file credential to custom scope
+  community.general.jenkins_credential:
+    id: "file-id"
+    type: "file"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    scope: "CUSTOM"
+    description: "File credential"
+    file_path: "../vars/my-secret.pem"
+
+- name: Add text credential to folder
+  community.general.jenkins_credential:
+    id: "text-id"
+    type: "text"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    description: "Text credential"
+    secret: "mysecrettext"
+    location: "folder"
+    url: "http://localhost:8080/job/test"
+
+- name: Add githubApp credential
+  community.general.jenkins_credential:
+    id: "githubapp-id"
+    type: "github_app"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    description: "GitHub app credential"
+    appID: "12345"
+    file_path: "../vars/github.pem"
+    owner: "github_owner"
+
+- name: Add sshKey credential
+  community.general.jenkins_credential:
+    id: "sshkey-id"
+    type: "ssh_key"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    description: "SSH key credential"
+    username: "sshuser"
+    file_path: "../vars/ssh_key"
+    passphrase: 1234
+
+- name: Add certificate credential (p12)
+  community.general.jenkins_credential:
+    id: "certificate-id"
+    type: "certificate"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    description: "Certificate credential"
+    password: "12345678901234"
+    file_path: "../vars/certificate.p12"
+
+- name: Add certificate credential (pem)
+  community.general.jenkins_credential:
+    id: "certificate-id-pem"
+    type: "certificate"
+    jenkins_user: "admin"
+    token: "{{ token }}"
+    description: "Certificate credential (pem)"
+    file_path: "../vars/cert.pem"
+    private_key_path: "../vars/private.key"
+"""
+RETURN = r"""
+details:
+    description: In case of errors return more details.
+    type: str
+    returned: Error
+token:
+    description:
+      - The generated API token if type is O(type=token).
+      - This is needed to authenticate API calls later.
+      - This should be stored securely, as it is the only time it is returned.
+    type: str
+    returned: success
+tokenUuid:
```

Comment:
As mentioned before (https://github.com/ansible-collections/community.general/pull/10170#discussion_r2105904597) you should use `snake_case`. This should be:
```suggestion
token_uuid:
```